tests/form/robot/ValidationTextBox.html

```diff
@@ -495,13 +495,39 @@
 							doh.robot.typeKeys('123', 1000, 600);
 							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
 								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
-								doh.is(undefined, this.textbox.get('value'), "get('value')");
+								doh.is(123, this.textbox.get('value'), "get('value')");
 								doh.f(this.textbox.isValid(), "!isValid()");
```

Comment:
I have updated the code to use the editOptions approach.  All tests pass.  

I took a look at combining the tests, however I couldn't get it to work.  If there is an example of combining "immediate" tests and tests in a "connected" event I can make the change.  The attempts I made caused the "already called!" error in the runner.js _check method.